soc/arm/st_stm32/stm32g4/soc.c

```diff
@@ -15,6 +15,11 @@
 #include <arch/cpu.h>
 #include <arch/arm/aarch32/cortex_m/cmsis.h>
 
+#if defined(PWR_CR3_UCPD_DBDIS)
```

Comment:
`PWR_CR3_UCPD_DBDIS` is defined for all G4 variants.
In that regard, this condition is superfluous.